entrypoint.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/bash
+#!/bin/sh
 
 # set CLI arguments
 cli_version="$1"
 paths="$2"
-parse_values="$3"
-display_level="$4"
-ansible_version="$5"
+project_id="$3"
+upload_values="$4"
+upload_metadata="$5"
+display_level="$6"
+ansible_version="$7"
 
 installDifferentSpotterCLIVersionIfNeeded() {
```

Comment:
This doesn't need to be exist since we're using a Docker image, right?

Reply:
True yes, we do use a Docker image for the CLI, but changing the CLI version cannot be applied by changing the CLI Docker image version dynamically, so if user wants different CLI we need to do it here.

But all in all maybe we don't even need this and can always use just the latest CLI version, @gberginc what do you think?

Reply:
Updates to use only the latest version.